packages/licenses/src/getPkgInfo.ts

```diff
@@ -0,0 +1,159 @@
+import path from "path";
```

Comment:
this won't pass linting. Run `pnpm lint` and see.

Reply:
I have ran `pnpm lint --fix` in this package